test/tests/functional/basic_single_server_auth_tests.js

```diff
@@ -8,7 +8,59 @@ var expect = require('chai').expect,
   Bson = require('bson');
 
 // Skipped due to use of topology manager
-describe.skip('Basic single server auth tests', function() {
+describe('Basic single server auth tests', function() {
+  it('should correctly authenticate server using scram-sha-256 using connect auth', {
+    metadata: { requires: { topology: 'auth', mongodb: '>=3.7.3' } },
+    test: function(done) {
+      const config = this.configuration;
+      const method = 'scram-sha-256';
+      const user = 'user';
+      const password = 'pencil';
+      const createUserCommand = {
+        createUser: user,
+        pwd: password,
+        roles: [{ role: 'root', db: 'admin' }],
+        digestPassword: true
+      };
+      const dropUserCommand = { dropUser: user };
+      const auth = [method, 'admin', user, password];
+
+      const createUser = cb => executeCommand(config, 'admin', createUserCommand, cb);
+      const dropUser = cb => executeCommand(config, 'admin', dropUserCommand, { auth }, cb);
+
+      const cleanup = err => {
+        executeCommand(config, 'admin', dropUserCommand, {}, () => done(err));
+      };
+
+      createUser((cmdErr, r) => {
+        expect(cmdErr).to.be.null;
+        expect(r).to.exist;
+
+        const server = new Server({
+          host: this.configuration.host,
+          port: this.configuration.port,
+          bson: new Bson()
+        });
+
+        server.on('connect', _server => {
+          dropUser((dropUserErr, dropUserRes) => {
+            expect(dropUserErr).to.be.null;
+            expect(dropUserRes).to.exist;
+
+            _server.destroy({ force: true });
+            expect(Object.keys(Connection.connections()).length).to.equal(0);
```

Comment:
I've been trying to remove all traces of these types of tests. This was here to ensure that all connections were really closed before moving on to the next test, and we have better ways of doing this now with the `ConnectionSpy` (at least). Ideally, `destroy` would take a callback but I just made [a ticket](https://jira.mongodb.org/browse/NODE-1443) for that

Reply:
Ok, I'll remove it.